### DISCUSSION TITLE
Collection of links to trailers and trucks which do not yet support autoload.

### DISCUSSION THREAD

**Discussion by SSlz** [ddewar/FS22_UniversalAutoloadModhubAddon#66] [Non-Modhub Mod Support]
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=243955&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=243382&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=238293&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=229132&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=233090&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=237555&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=249709&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=243774&title=fs2022

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=250565&title=fs2022

these links are for trailers and trucks that I think deserve an autoload feature.

I would be happy if they could get an autoload function.

Many Thanks

**Comment by ddewar** [comment]
Hey @SSlz thanks for this collection of links, these all look like great options for autoload. Any chance you would open up an issue on the project? It will help me prioritize the work for it. 

https://github.com/ddewar/FS22_UniversalAutoloadModhubAddon/issues

**Comment by Wingman93** [comment]
[This Here](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=270264&title=fs2022) and [This Here](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=261023&title=fs2022) when its possible @ddewar

**Reply by ddewar** [reply]
Hey @Wingman93, these look like they can be added and are hosted on the Modhub? If so, please open up a new issue on the GitHub repo and then I can get it prioritized. Sometimes the mods here get lost in the thread of other requests.

**Reply by Wingman93** [reply]
@ddewar these 2 Mods are in the Modhub official Released :)
